src/extension-runners/index.js

```diff
@@ -0,0 +1,299 @@
+/* @flow */
+
+import readline from 'readline';
+import tty from 'tty';
+
+import type Watchpack from 'watchpack';
+
+import type {
+  IExtensionRunner,  // eslint-disable-line import/named
+  ExtensionRunnerReloadResult,
+} from './base';
+import {
+  showDesktopNotification as defaultDesktopNotifications,
+} from '../util/desktop-notifier';
+import {createLogger} from '../util/logger';
+import type {FileFilterCreatorFn} from '../util/file-filter';
+import {
+  createFileFilter as defaultFileFilterCreator,
+} from '../util/file-filter';
+import defaultSourceWatcher from '../watcher';
+import type {OnSourceChangeFn} from '../watcher';
+
+
+const log = createLogger(__filename);
+
+
+export type MultiExtensionRunnerParams = {|
+  runners: Array<IExtensionRunner>,
+  desktopNotifications: typeof defaultDesktopNotifications,
+|};
+
+// Export everything exported by the firefox-desktop runner.
+export * from './firefox-desktop';
```

Comment:
oh, sorry, I totally read this backwards. I thought you were exporting all of `src/extension-runners` from `src/extension-runners/firefox-desktop` which didn't seem useful to me.

Ship it! I agree that exporting everything from `src/extension-runners` to conceal the sub-module implementation is a useful pattern.